app/assets/src/api/index.js

```diff
@@ -15,8 +15,8 @@ const postWithCSRF = async (url, params) => {
     // Just return the data.
     // resp also contains headers, status, etc. that we might use later.
     return resp.data;
-  } catch (e) {
-    return Promise.reject(e.response.data);
+  } catch (error) {
+    return Promise.reject(error);
```

Comment:
I know the new upload flow uses this so you might want to check downstream to make sure they're OK

Reply:
Reverted.
Do not really need the change.